src/components/table-transactions/table-transactions.scss

```diff
@@ -0,0 +1,47 @@
+.table-transactions{
```

Comment:
are your pre-commit hooks working? there should be a space between classname and `{`